test/Kozani/KozaniManagerRuntimeTests/KozaniManagerRuntimeTests.cpp

```diff
@@ -18,18 +76,37 @@ namespace Test::KozaniManagerRuntimeTests
         TEST_CLASS_SETUP(ClassSetup)
         {
             ::TB::Setup();
+            ::TP::AddPackage(KozaniManagerTestPackage::c_PackageDirName, KozaniManagerTestPackage::c_PackageFullName);
             return true;
         }
 
         TEST_CLASS_CLEANUP(ClassCleanup)
         {
+            ::TP::RemovePackageIfNecessary(KozaniManagerTestPackage::c_PackageFullName);
             ::TB::Cleanup();
             return true;
         }
 
-        // TODO Replace this with your own tests
-        TEST_METHOD(JustDoIt)
+        TEST_METHOD(ActivateRemoteApplicationTests)
         {
+            try
+            {
+                winrt::Microsoft::Kozani::ManagerRuntime::ManagerRuntimeManager runtimeManager;
+                auto statusCallback{ winrt::make_self<MyKozaniStatusCallback>() };
+
+                runtimeManager.ActivateRemoteApplication(winrt::Windows::ApplicationModel::Activation::ActivationKind::Launch,
+                    L"Microsoft.WindowsCalculator_8wekyb3d8bbwe!App", L"c:\\data\\connection.rdp", L"c:\\data\\additionalSettings.txt", 
```

Comment:
I assume the hardcoded pathnames are temporary hacks soon to be replaced...

SUGGESTION: Create `connection.rdp` and `additionalSettings.txt` in the project directory, tell VS to copy them to OutDir on build. See https://stackoverflow.com/questions/18743907/visual-studio-how-to-copy-to-output-directory-without-copying-the-folder-stru for more details, but with Content instead of ContentWithTargetPath if the data files are in the project's root (next to the .vcxproj)

P.S. `PreserveNewest` will update the OutDir copy if the source is newer, `Always` copies it every time you build, `Never` obviously never does. `PreserveNewest` is usually what you'd want.

Reply:
yes, it is temporary to get through the API testing. No real functionality implemented yet so just a string to pass through will be enough. When it is implemented we will change the test to do meaningful work.